README.md

```diff
@@ -57,9 +57,48 @@ This driver supports only one topology key:
 that represents availability by zone.
 
 ## Kubernetes User Guide
-
 ### Install Driver
 
+#### Preflight (Ubuntu like)
+1. You need to install golang
+```
+sudo apt update && sudo apt install golang
```

Comment:
Instead of providing a distribution specific method of installing go-lang here (as it could be different for different environments). Lets just link to the official golang website and ask users to follow instructions for their distribution from there